guides/manual/1-overview.md

```diff
@@ -1,71 +1,151 @@
-decades.
-
-<br>
-
-## Why WarpDrive?
+## Overview
 
 WarpDrive is the data framework for building ambitious applications.
 
-What do we mean by ambitious? WarpDrive is ideal for applications looking to
-be best-in-class: whether that's a small todo app, e-commerce, a
-social app, or an enterprise b2b software solution.
+What do we mean by ambitious? WarpDrive is ideal for applications looking to be best-in-class: whether that's a small todo app, e-commerce, a social app, or an enterprise b2b software solution.
+
+The capabilities WarpDrive provides are designed to seamlessly handle and simplify the hardest parts of state management when building an app, helping you focus on creating the features and user experiences that drive value.
+
+Its innovative approach to [fine grained reactivity](https://dev.to/ryansolid/a-hands-on-introduction-to-fine-grained-reactivity-3ndf) enables rapidly developing robust, performant web applications using any [Signals](https://github.com/tc39/proposal-signals#readme) compatible framework such as [Ember](https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/), [Svelte](https://svelte.dev/docs/svelte/what-are-runes), [Angular](https://angular.dev/guide/signals), [Vue.js](https://vuejs.org/guide/extras/reactivity-in-depth.html), [SolidJS](https://www.solidjs.com/tutorial/introduction_signals),
+[Preact](https://preactjs.com/guide/v10/signals/) or [Lit](https://lit.dev/docs/data/signals/).
+
+```hbs
+  Hello {{@user.name}}!
+```
+
+By building around the same interface as the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), WarpDrive makes powerful request management features like caching, deduping and data normalization feel simple to use. We pair this with [reactive control flow](./concepts/reactive-control-flow.md) to give apps the ability to declaratively derive states with safety.
+
+```glimmer-ts
+import { Request } from '@warp-drive/ember';
+import { findRecord } from '@ember-data/json-api/request';
+import { Spinner } from './spinner';
+
+export default <template>
+  <Request @query={{findRecord "user" "1"}}>
+    <:error as |error state|>
+      <h2>Whoops!</h2>
+      <h3>{{error.message}}</h3>
+      <button {{on "click" state.retry}}>Try Again?</button>
+    </:error>
+
+    <:content as |result|>
+      Hello {{result.data.name}}!
+    </:content>
+
+    <:loading as |state|>
+      <Spinner @progress={{state.completedRatio}} />
+      <button {{on "click" state.abort}}>Cancel?</button>
+    </:loading>
+  </Request>
+</template>
+```
+
+Web clients are like high-latency, remotely distributed, often-stale partial replicas of server state - but requests using WarpDrive utilize an [advanced relational cache](./5-caching.md) that simplifies these problems, solving them when it can and providing intelligent escape valves for when it can't so that no matter what you can quickly get the data you need in the right state.
```

Comment:
This is a pretty important, but very dense paragraph. I wonder if we should illustrate it more clearly for less experienced devs. (Will continue this thought after my computer restarts to install upgrades...)